README.md

```diff
@@ -111,10 +111,13 @@ I am not convinced that this is the best strategy so this may well change in a f
 If you like to apply different styles to different tags, this is now possible
 (thank you @darthmachina).  Tags now have a class that reflects the tag name.
 So if you have a tag `#foo/bar` you will be able to style it with your favorite
-color for foobars using:
+color for foo/bars (which just has to be HotPink) using:
```

Comment:
Not sure if these have to resolved or I can just comment, but I had to put in my love for using HotPink here!

Reply:
😄